_pages/history.md

```diff
@@ -0,0 +1,100 @@
+implementation than it was for Selenium users. 
+
+When Jari Bakken, a Watir developer, was exposed to WebDriver in 2009, he began developing a Ruby implementation of it 
+for both Selenium and Watir. Jari had already written [Celerity](https://github.com/jarib/celerity), 
+a JRuby project to implement a wrapper for HtmlUnit, to provide a headless testing option with the same API as Watir. 
+With his work on watir-webdriver, he decided to take the Celerity specs and make them usable for both 
+projects.([7](https://github.com/watir/watirspec/tree/9443ea77ab998b4f676a120221b460dcab421933)). 
+In doing so he created the WatirSpec repository which could be used by any of the Ruby-based projects that 
+implemented the Watir API.
+
+The functionality being provided by WebDriver caused the Watir team to add features and change its conventions 
+when it made sense to do so. The big backward-incompatible change in the [Watir 2.0](http://watir.com/watir-2-0/) 
+(released in August of 2011) was the move from one-indexing to zero-indexing. 
+Indexing “like a person” makes sense until you have to constantly figure out if you are working with a 
+Watir Collection or a Ruby Array (or with WebDriver). The challenge of bringing Watir into complete 
+compliance with the WatirSpec was taken up by  Jarmo Pertman and culminated in the release 
+of [Watir 3.0](http://watir.com/watir-3-released/) in April 2012.
+
+To encourage the use of watir-webdriver, Hugh McGowan renamed the Watir code base “watir-classic” and 
+released Watir 4.0 in September 2012 as a [meta-gem](https://github.com/watir/watir_meta#about). 
+This release defaulted to using the watir-classic implementation for Internet Explorer and using the 
+watir-webdriver implementation for all other browsers.
+
+In January of 2015 Jari decided to step down as maintainer of the selenium-webdriver and watir-webdriver
+gems.(<strong>[7](https://groups.google.com/forum/#!msg/selenium-developers/h2Ie4FNHmq4/OBsaatq5y6gJ)</strong>).
+Alex Rodionov had been an active contributor for the previous three years and took over primary ownership of
+the project with assistance from Titus Fortner who was working with Bret at Blackbaud at that time
+ and had just been officially added to the development team.
+
+When the W3C started creating a standard [specification for WebDriver](http://w3c.github.io/webdriver/webdriver-spec.html),
+ all of the major browser vendors got on board to implement a driver for their respective browsers. 
+ Additionally, Microsoft released the Edge browser to replace Internet Explorer.
+  The combination of these things made the original implementation of
+ Watir (watir-classic) increasingly obsolete. As such, in 2016 Titus spearheaded a move for
+ Watir to rebrand itself, deprecate both watir-classic and watir-webdriver, 
+ and release Watir 6.0 with an updated version of the Selenium-backed code. Based on numerous conversations
```

Comment:
What happened to Watir 5?

Reply:
Watir 5 was to support watir-classic 4, though I'm not sure why the meta-gem needed a version bump for that. I'm also somewhat unclear on what prompted the release of watir-classic 4. @jarmo can you clarify this part? was it the change from `Watir::IE` to `Watir::Browser` being backward incompatible?

Reply:
If my memory isn't lying and looking at the watir-classic changelog then watir-classic was bumped to version 4 just because it had few backwards incompatible API changes. That release didn't have any major changes in it.

If I'm not mistaken then watir version 5 was the first release where watir gem was made to be a so-called meta-gem, which allowed to use watir-classic or watir-webdriver depending on the configuration. Original watir was renamed to watir-classic. This change was mostly done for everyone using watir-classic to be able to switch between watir-webdriver easily so that at one point watir-classic could be deprecated and watir-webdriver would be default. It also allowed to select browser and if IE was used, then by default watir-classic was started, but if Chrome or Firefox was specified, then watir-webdriver was loaded under the hood. At that time watir-webdriver IE driver had problems and was a lot slower than watir-classic implementation so it made sense to still fallback to watir-classic for IE. Watir 6 was basically a rename of watir-webdriver to watir and deprecating meta-gem and watir-classic, right?

Reply:
@titusfortner is https://github.com/watir/watir.github.io/pull/125#discussion_r207790285 enough for adding information regarding Watir 3-5 for you?

Reply:
I think what we have right now is good as far as the code part of the Watir story. Thanks for your assistance.